modules/cli/src/main/scala/scala/cli/commands/Uninstall.scala

```diff
@@ -0,0 +1,35 @@
+package scala.cli.commands
+
+import caseapp._
+
+import scala.io.StdIn.readLine
+import scala.cli.CurrentParams
+
+object Uninstall extends ScalaCommand[UninstallOptions] {
+  def run(options: UninstallOptions, args: RemainingArgs): Unit = {
+    CurrentParams.verbosity = options.verbosity.verbosity
+    val binDirPath =
+      options.binDirPath.getOrElse(scala.build.Directories.default().binRepoDir / "scala-cli")
+    val destBinPath = binDirPath / options.binaryName
+
+    if (!options.force) {
+      println("Do you want to uninstall scala-cli [Y/n]")
```

Comment:
You can use interactive functions, it could be something similar as [here](https://github.com/VirtusLab/scala-cli/blob/d8af06401ab9aaf63eef5e95e0e1c0909d4ad4e5/modules/cli/src/main/scala/scala/cli/commands/Update.scala#L62)